packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -52,6 +52,7 @@ export default class MainProcess {
       height,
       title: 'Teleport Terminal',
       icon: getAssetPath('icon.png'),
+      backgroundColor: '#01172C',
```

Comment:
What do you think about making `getPlatform` compatible with Electron and then doing this?

```suggestion
      backgroundColor: theme.colors.primary.main,
```


`packages/design/src/theme/utils/getPlatform.js`:

```typescript
export default function getPlatform() {
  if (typeof window !== 'undefined') {
    const userAgent = window.navigator.userAgent;

    return {
      isWin: userAgent.indexOf('Windows') >= 0,
      isMac: userAgent.indexOf('Macintosh') >= 0,
      isLinux: userAgent.indexOf('Linux') >= 0,
    };
  } else {
    const platform = process.platform;
    const isWin = platform === 'win32';
    const isMac = platform === 'darwin';

    return {
      isWin,
      isMac,
      isLinux: !isWin && !isMac,
    };
  }
}
```

Reply:
Good advice, it will make our life easier when updating the theme.